### PR TITLE
Change file, function and references in prep of winsorization process

### PR DESCRIPTION
**Pull request by garypolzin**
# Description
Previously the filters were all named and designed to 'null outliers' but this process now winsorizes so no values are nulled, changed names accordingly

[Branch run](https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:pre-winsorization-rename-Ind-CQC-Filled-Post-Estimates-Pipeline:8e74c177-3243-43e9-840c-32c630783ad9)

### CODE REVIEW

File: utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/null_filled_posts_where_locations_use_invalid_missing_data_code.py [hunk 1]
```diff
@@ -17,15 +17,17 @@ def null_filled_posts_where_locations_use_invalid_missing_data_code(
     df: DataFrame,
 ) -> DataFrame:
     """
-    Null rows where locations have 999 in ascwds_filled_posts_dedup_clean.
+    Null ascwds_filled_posts_dedup_clean values where locations have 999 in ascwds_filled_posts_dedup_clean.
 
-    '999' is used elsewhere in ASCWDS to represent not known. Whilst that isn't the case for filled posts (i.e. the raw variables total staff), upon investigation, all 999 entries at the time of investigation do not look to be an accurate reflection of that location's staff so we are removing them for data quality.
+    '999' is used elsewhere in ASCWDS to represent not known. Whilst that isn't the case for the variable totalstaff,
```
Comment (MarionJHolloway) [inline utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/null_filled_posts_where_locations_use_invalid_missing_data_code.py:22]: There are doc string linters available. I will add a ticket to the trello board to install a suitable one and I can do it on a quiet day :)

File: utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/winsorize_care_home_filled_posts_per_bed_ratio_outliers.py [hunk 2]
```diff
@@ -18,27 +18,33 @@ class NumericalValues:
...
+    Outlier detection:
+        The number of beds at each location are banded into categorical groups and the average 'filled post per bed' ratio
+        in each band is used to determine the 'expected filled posts' for each location. The residuals (the difference
+        between actual and expected filled posts) are calculated, followed by the standardised residuals (residuals divided
+        by the squart root of the filled post figure). The values at the top and bottom end of the standarised residuals
+        are deemed to be outliers. The proportion of data to be identified as outliers is determined by the value of
+        PERCENTAGE_OF_DATE_TO_REMOVE_AS_OUTLIERS.
+
+    Winsorization:
+        Filled post figures deemed outliers will be replaced by less extreme values calculated during the winsorization
```
Comment (MarionJHolloway) [inline utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/winsorize_care_home_filled_posts_per_bed_ratio_outliers.py:40]: Are we cleaning only the filled posts figure or the beds ratio as well? I think we probably want to do the same across all the filters for consistency.

**Review by MarionJHolloway — APPROVED** [review]
Looks good - much easier to review thanks!

File: utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/null_filled_posts_where_locations_use_invalid_missing_data_code.py (hunk 1 above)
Comment (garypolzin) [reply utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/null_filled_posts_where_locations_use_invalid_missing_data_code.py:22]: ah ok, I changed the text a bit but I often have my vscode on half screen hence the shorter width!

File: utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/winsorize_care_home_filled_posts_per_bed_ratio_outliers.py (hunk 2 above)
Comment (garypolzin) [reply utils/ind_cqc_filled_posts_utils/clean_ascwds_filled_post_outliers/winsorize_care_home_filled_posts_per_bed_ratio_outliers.py:40]: there is a direct relationship between posts/ratio so they need to be aligned

we generally filter filled posts, and then recalculate `ratio = posts / beds` after

**Pull request merged**